app/assets/javascripts/rentals/new.js.erb

```diff
@@ -81,6 +81,16 @@ $(document).ready(function () {
     costCalculation();
   });
 
+  $('#rentals-new-toc').change(function() {
+    var button = document.getElementById('rentalSubmit')
```

Comment:
could select using jquery if you want seeing as we already do that to get the onchange event.

Reply:
I was just being extra careful, might change it once everything starts working again.